doc/source/how-to/documenting.rst

```diff
@@ -469,349 +470,115 @@ branch within this repository. This is done by uploading the generated
+    
+    doc-deploy-dev:
+      name: "Deploy developers documentation"
+      if: github.event_name == 'push'
+      runs-on: ubuntu-latest
+      needs: doc-build
+      steps:
+        - name: "Deploy the latest documentation"
+          uses: pyansys/actions/doc-deploy-dev@v1
+          with:
+              cname: "<library>.docs.pyansys.com"
+              token: ${{ secrets.GITHUB_TOKEN }}
+    
+    doc-deploy-stable:
+      name: "Deploy stable documentation"
+      if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+      runs-on: ubuntu-latest
+      needs: doc-deploy-dev
+      steps:
+        - name: "Deploy the stable documentation"
+          uses: pyansys/actions/doc-deploy-stable@v1
+          with:
+              cname: "<library>.docs.pyansys.com"
+              token: ${{ secrets.GITHUB_TOKEN }}
+
+
+Deploying to ``gh-pages`` of another repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are planning to deploy documentation to a repository other than
+the one for your project, make sure you create this new repository before deploying
+your documentation for the first time.
+
+.. warning::
+
+    Deploying your documentation to another repository is discouraged. It
+    translates to more mainteinance work and does not support the multi-version
+    documentation.
+
+For deploying the documentation to another repository, use the following workflow:
 
 .. code-block:: yaml
 
-    if: github.ref == 'refs/heads/main'
-
+    doc-deploy:
+      name: "Deploy documentation to a different repo"
+      runs-on: ubuntu-latest
+      needs: doc-build
+      steps:
+        - name: "Deploy documentation"
+          uses: pyansys/actions/doc-deploy-to-repo@main
+          with:
+            cname: "<library>.docs.pyansys.com"
+            repository: "<owner>/<repository-name>"
+            bot-id: ${{ secrets.BOT_APPLICATION_ID }}
+            bot-token: ${{ secrets.BOT_APPLICATION_PRIVATE_KEY }}
 
 Access online documentation
 ---------------------------
 Documentation for the latest stable release of a PyAnsys library is accessible
-from its repository. You can generally access the latest development version of
-the documentation tracking the ``main`` branch by adding the prefix ``dev.`` to
-the URL for the latest stable release.
+from its repository. The canonical name for the documentation of the project is
+constructed using the following structure:
+
+``https://<product>.docs.pyansys.com``
+
+You can generally access the latest development version of the documentation by
+adding the prefix ``dev.`` to the URL for the latest stable release.
+
+.. warning::
+
+    PyAnsys projects support now multi-version documentation, meaning that
+    stable and development versions are collected under the same webpage. A
+    drop-down button for selecting desired version should be available in the
+    top right corner of the navigation bar in the documentation page.
```

Comment:
it will be available on if you setup the ``"navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],`` in conf file otherwise it will be on top left of the page.

Reply:
Can't we force all this in the ansys-sphinx-theme?

Reply:
yes. i was looking that right now. we can do that.

Reply:
If we could force this directly from our theme it would be awesome!

Reply:
Totally... I mean, I'm fine adding it in one repo.. but if we have to add it in 3 or more... please add it to the main theme 😄